app/controllers/telegram_webhooks_controller.rb

```diff
@@ -97,4 +66,108 @@ def help(*)
   def action_missing(action)
     reply_with :message, text: "Can not perform #{action}" if command?
   end
-end
+
+  private
+  def getArgsFromCommand(* args)
```

Comment:
@filipecosta01 Is there any specific reason for this space between the star symbol and the args?